app/models/nested_geo_bbox.rb

```diff
@@ -0,0 +1,31 @@
+class NestedGeoBbox < ActiveTriples::Resource
+  property :label, predicate: ::RDF::Vocab::DC.coverage
```

Comment:
All of the predicates have changed, so the new combined :nested_geo property should have these properties:

- label	dct:title
- point	https://purl.org/geojson/vocab#coordinates
- bbox	https://purl.org/geojson/vocab#bbox